docs/source/user_guide/cli.rst

```diff
@@ -4,8 +4,7 @@
 Command Line Interface
 ======================
 
-.. automodule:: easylink.cli
-
 .. click:: easylink.cli:easylink
    :prog: easylink
-   :show-nested:
+   :nested: full
+   :commands: run, generate-dag
```

Comment:
I do this to control the order of cli command docs (otherwise it's alphabetical but I want `run` to come before `generate-dag` since it's the more important one.